lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -36,26 +41,43 @@ def _batchify(iterable, n=1):
         yield iterable[i : min(i + n, length)]
 
 
+ARYN_DETR_MODEL = "Aryn/deformable-detr-DocLayNet"
+DEFAULT_ARYN_PARTITIONER_ADDRESS = "https://api.aryn.cloud/v1/document/partition"
+_TEN_MINUTES = 600
+
+
+class ArynPDFPartitionerException(Exception):
+    def __init__(self, message, can_retry=False):
+        super().__init__(message)
+        self.can_retry = can_retry
+
+
+def _can_retry(e: BaseException) -> bool:
+    if isinstance(e, ArynPDFPartitionerException):
+        return e.can_retry
+    else:
+        return False
+
+
 pdf_miner_cache = DiskCache(os.path.join(tempfile.gettempdir(), "SycamoreCache/PDFMinerCache"))
 
 
-class SycamorePDFPartitioner:
+class ArynPDFPartitioner:
```

Comment:
I would think this belongs in the aryn_partitioner file as we'd want to abstract the detr implementation. But I guess its coupled currently so no harm

Reply:
aryn_partitioner.py gets deleted with this pr